alf/algorithms/ddpg_algorithm.py

```diff
@@ -277,7 +279,8 @@ def actor_loss_fn(dqda, action):
             loss = loss.sum(list(range(1, loss.ndim)))
             if self.action_l2 > 0:
                 assert action.requires_grad
-                loss += self.action_l2 * torch.norm(action)
+                loss += self.action_l2 * (torch.mean(
+                    torch.norm(action, dim=-1)**2))
```

Comment:
In case that action is multi-dimensional: `torch.norm(list(range(1, action.ndim))`